lib/ash_phoenix/live_view.ex

```diff
@@ -656,10 +656,15 @@ defmodule AshPhoenix.LiveView do
   # => %Phoenix.LiveView.Socket{assigns: %{results: [1,2,3], page: %Ash.Page.Offset{results: nil}}}
   ```
   """
-  @spec assign_page_and_stream_result(Phoenix.LiveView.Socket.t(), Ash.Page.page(), atom()) ::
+  @spec assign_page_and_stream_result(
+          Phoenix.LiveView.Socket.t(),
+          Ash.Page.page(),
+          atom(),
+          atom()
+        ) ::
           Phoenix.LiveView.Socket.t()
-  def assign_page_and_stream_result(socket, page, result_key \\ :results) do
+  def assign_page_and_stream_result(socket, page, result_key \\ :results, page_key \\ :page) do
```

Comment:
Okay, one last piece of feedback, lets make this an options list 😄 I've learned to be wary of multiple optional arguments.

Reply:
i was thinking about that!

alright, i'll give it a try.